docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+version: '3.6'
+services:
+  db:
+    image: postgis/postgis
+    container_name: ev-charger-app-db
+    environment:
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_DB: db
+    ports:
+      - target: 5432
+        published: 5432
+        protocol: tcp
+        mode: host
+    volumes:
+      - db_persdata:/var/lib/postgrespers
+    networks:
+      - backdb
+    expose:
+      - 5432
+
+  backend:
+    image: norberttalpos/ev-charger-app-backend:latest
+    container_name: ev-charger-app-backend
+    environment:
+      WAIT_HOSTS: postgres:5432
+      SPRING_DATASOURCE_URL: jdbc:postgresql://db:5432/db
+      SPRING_DATASOURCE_USERNAME: postgres
+      SPRING_DATASOURCE_PASSWORD: postgres
+      SPRING_FLYWAY_URL: jdbc:postgresql://db:5432/db
+      SPRING_FLYWAY_USER: postgres
+      SPRING_FLYWAY_PASSWORD: postgres
+      JWT_HEADER: Authorization
+      SPRING_JPA_DATABASE_PLATFORM: org.hibernate.spatial.dialect.postgis.PostgisDialect
+      SPRING_MAIL_HOST: smtp.gmail.com
+      SPRING_MAIL_PORT: 587
+      SPRING_MAIL_USERNAME: ev.charger.app@gmail.com
+      SPRING_MAIL_PASSWORD: jokerdes
```

Comment:
hmmm

Reply:
The credentials are to be added later privately. No problem for me